src/sage/combinat/posets/hasse_cython_flint.pyx

```diff
@@ -0,0 +1,148 @@
+r"""
```

Comment:
Shouldn't you add `# cython: binding=True` ?

Reply:
answered in https://github.com/sagemath/sage/pull/35741#issuecomment-1581226610